systemd/coreos-installer-reboot-loaddev.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Reboot from zVM LOADDEV after CoreOS Installer
+ConditionVirtualization=zvm
+ConditionPathExists=/run/coreos-installer-loaddev
+
+Requires=coreos-installer.target
+After=coreos-installer.target
+After=coreos-installer-reboot.service
```

Comment:
Hmm, wouldn't we have already issued a `systemctl reboot` if we run after `coreos-installer-reboot.service`?

I think it'd be clearer if we didn't have both services run.

This service also currently ignores `skip_reboot`.

Let's instead have both of them fire off of `/run/coreos-installer-reboot` and add `ConditionKernelCommandLine=!zvm` in `coreos-installer-reboot.service`?

Then we can get rid of `/run/coreos-installer-loaddev`.